docs/get-started/bridge/shimmerbridge.md

```diff
@@ -0,0 +1,230 @@
+
+- [AVAX](https://explorer.evm.shimmer.network/address/0xEAf8553fD72417C994525178fC917882d5AEc725?tab=contract) - Avalanche
+Address: `0xEAf8553fD72417C994525178fC917882d5AEc725`
+
+- [FTM](https://explorer.evm.shimmer.network/address/0x8C96Dd1A8B1952Ce6F3a582170bb173eD591D40D?tab=internal_txns) - Fantom 
+Address:`0x8C96Dd1A8B1952Ce6F3a582170bb173eD591D40D`
+
+- [BNB](https://explorer.evm.shimmer.network/address/0x2A6F394085B8E33fbD9dcFc776BCE4ed95F1900D?tab=contract) - Binance Coin
+Address: `0x2A6F394085B8E33fbD9dcFc776BCE4ed95F1900D`
+
+:::
+
+### **LayerZero ShimmerBridge wrapped tokens on connected networks**
+
+#### **Ethereum**
+:::info
+
+- USDC - USD Coin
+Address: `0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48`
+
+- USDT - Tether USD
+Address: `0xdAC17F958D2ee523a2206206994597C13D831ec7` 
+
+- WBTC - Wrapped Bitcoin
+Address: `0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599`
+
+:::
+
+#### **Arbitrum**
+:::info
+
+- USDC - USD Coin
+Address: `0xaf88d065e77c8cC2239327C5EDb3A432268e5831`
+
+- USDT - Tether USD
+Address: `0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9`
+
+- WBTC - Wrapped Bitcoin
+Address: `0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f`
+
+- WETH - Wrapped Ether
+Address: `0x82af49447d8a07e3bd95bd0d56f35241523fbab1`
+
+:::
+
+#### **Polygon**
+:::info
+
+- USDC - USD Coin
+Address: `0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359`
+
+- USDT - Tether USD
+Address: `0xc2132D05D31c914a87C6611C10748AEb04B58e8F`
+
+- WBTC - Wrapped Bitcoin
+Address: `0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6`
+
+- WETH - Wrapped Ether
+Address: `0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619`
+
+:::
+
+#### **Optimism**
+
+:::info
+
+- USDC - USD Coin
+Address: `0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85`
+
+- USDT - Tether USD
+Address: `0x94b008aA00579c1307B0EF2c499aD98a8ce58e58`
+
+- WBTC - Wrapped Bitcoin
+Address: `0x68f180fcCe6836688e9084f035309E29Bf0A2095`
+
+- WETH - Wrapped Ether
+Address: `0x4200000000000000000000000000000000000006`
+
+:::
+
+#### **Binance Smart Chain**
+
+:::info
+
+- BNB - Binance Coin
+Native Gas Token
+
+- USDC - USD Coin
+Address: `0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d`
+
+- USDT - Tether USD
+Address: `0x55d398326f99059fF775485246999027B3197955`
+
+:::
+
+#### **Base**
+:::info
+
+- USDC - USD Coin
+Address: `0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913`
+
+- WETH - Wrapped Ether
+Address: `0x4200000000000000000000000000000000000006`
+
+:::
+
+#### **Fantom**
+:::info
+
+- FTM - Fantom
+Native Gas Token
+
+- USDC - Layer Zero USD Coin
+Address: `0x28a92dde19D9989F39A49905d7C9C2FAc7799bDf`
+
+:::
+
+#### **Avalanche C-Chain**
+:::info
+
+- AVAX - Avalanche
+Native Gas Token
+
+- USDC - USD Coin
+Address: `0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E`
+
+- USDT - Tether USD
+Address: `0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7`
+
+:::
+
+
+
+---
+
+
+**The Open Source code of the ShimmerBridgeA Web Application is available [here]()**
+**The Audit Report of ShimmerBridge Web Application is available [here]()**
```

Comment:
Please use something like "on the Zokyo Website" or "on GitHub: for the links. Could we add dummy links to test the build?